js/ReadMe.md

```diff
@@ -1,24 +1,112 @@
-# JS Wrapper for INJI VC Renderer : 
+### InjiVcRenderer - Js Library
 
+#### Features
+- Downloads the SVG template from the renderMethod field of the VC to support VC Data Model 2.0.
+- Replace the placeholders in the SVG template with actual VC Json Data.
+
+#### Running the Example App
+
+**Steps to run example app**
+- Run `cd js`and go into js folder
```

Comment:
run cd js and go into js folder, both are same thing right? cd js means navigate to js folder

Reply:
Yeah, I will update it.